File: Codeforces/Newcomers Training/Sheet #1 (Data type - Conditions)/L.cpp
#include <bits/stdc++.h>
using namespace std;
 
//Short code
typedef long long ll;
typedef vector<int> vi;
typedef pair<int,int> pi;
 
//Macros
#define F first
#define S second
#define pb push_back
#define mp make_pair
#define loop(i,a,b) for (int i = a; i < b; i++)
 
int main ()
{
  //freopen ("input.txt","r",stdin);
  //freopen ("output.txt","w",stdout);
 
  /* Code here */
  int a,b,c,d;
  cin>> a>>b>>c>>d;
  cout<< min(min(a,b),min(c,d)) <<" "<< max(max(a,b),max(c,d));
  //fclose (stdin);
  //fclose (stdout);
  return 0;
}

File: Codeforces/Newcomers Training/Sheet #3 (Arrays)/L.py
n = int(input())
arr = list(map(int, input().split()))
 
M = max(arr)
m = min(arr)
for i in arr:
	if i == M:
		print(m, end = " ")
	elif i == m:
		print(M, end = " ")
	else:
		print(i, end = " ")
